DSharpPlus/Entities/DiscordChannel.cs

```diff
@@ -279,6 +283,31 @@ public Task<DiscordMessage> SendMultipleFilesAsync(Dictionary<string, Stream> fi
             return this.Discord.ApiClient.UploadFilesAsync(this.Id, files, content, tts, embed, mentions);
         }
 
+        /// <summary>
+        /// Sends a Message.
```

Comment:
Please reuse the same documentation as older methods.

Reply:
This is completed.